cmd/cmd.go

```diff
@@ -0,0 +1,88 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+package cmd
+
+import (
+	. "github.com/microsoft/go-sqlcmd/cmd/commander"
+	"github.com/microsoft/go-sqlcmd/cmd/helpers"
+	"github.com/microsoft/go-sqlcmd/cmd/helpers/output"
```

Comment:
should helpers move from /cmd to /internal?
Doing so would hide public types from other apps that reference this repo. Let's be deliberate about what goes public and what does not. 

I think it's certainly desirable for third parties to reference /pkg/sqlcmd, for example.

Reply:
Done in next iteration